src/css/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin flexbox($direction, $wrap, $alignContent, $justify, $alignItems) {
+  display: flex;
```

Comment:
Such a great idea for a mixin! I love that it's all together too rather than being in multiple mixins